src/analyze-trace-file.ts

```diff
@@ -30,9 +31,9 @@ const importExpressionThreshold = 10;
 
 const reportHighlights = argv.json ? reportJson : reportText;
 
-reportHighlights(tracePath, argv.expandTypes ? typesPath : undefined, thresholdDuration, minDuration, minPercentage, importExpressionThreshold).then(found => process.exit(found ? 0 : 1)).catch(err => {
+reportHighlights(tracePath, argv.expandTypes ? typesPath : undefined, thresholdDuration, minDuration, minPercentage, importExpressionThreshold).then(found => process.exitCode = found ? 0 : 1).catch(err => {
```

Comment:
I'm curious, why not

```suggestion
reportHighlights(tracePath, argv.expandTypes ? typesPath : undefined, thresholdDuration, minDuration, minPercentage, importExpressionThreshold)
  .then(found => exit(found ? 0 : 1))
  .catch(err => {
```

to be consistent?

Reply:
I was trying to follow the guidance in the node docs that you shouldn't call exit unless you really mean it.  I don't have a strong preference one way or the other.